spec/spec_helper.rb

```diff
@@ -17,7 +17,8 @@
 ACCOUNT_REFS  = [ :accounts,
                   :account,
                   :account_history,
-                  :account_holds ]
+                  :account_holds,
+                  :position ]
```

Comment:
did you mean to have a space after position?

Reply:
no. That is a mistake. good catch